site/tutorials/snippets/17/install/install-full.md

```diff
@@ -0,0 +1,50 @@
+
+## Install Keptn in your cluster
+Duration: 5:00
+
+To install the latest release of Keptn with full _quality gate + continuous delivery capabilities_ in your Kubernetes cluster, execute the `helm install` command.
+
+```
+helm install keptn keptn/keptn -n keptn --version=0.17.0 --create-namespace --set=continuous-delivery.enabled=true
```

Comment:
The helm install should probably contain a `--wait` to block while everything is getting ready